cmd/kev/cmd/dev.go

```diff
@@ -139,8 +131,8 @@ func verifySkaffoldExpectedFlags(cmd *cobra.Command) error {
 		}
 
 		if len(kevenv) == 0 {
-			log.Warnf("Skaffold will use profile pointing at default `%s` environment. You may override it with `--kev-dev` flag.", envs[0])
-			cmd.Flag("kev-env").Value.Set(envs[0])
+			log.Warnf("Skaffold will use profile pointing at default `%s` environment. You may override it with `--kev-dev` flag.", "dev")
```

Comment:
we could get the name of sandboxed environment here so that if ever change it we'll have up to date messages. Same goes for the line below.

Reply:
That's in the upcoming `--skaffold` focused PR.